### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by D1M0H41K**

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def from_json(json_object):
                     json_object['done'],
                     datetime.datetime.fromisoformat(json_object['date']),
                     json_object['id'])
-    except KeyError:
+    except KeyError and ValueError:
```
Comment (moznuy) [inline db.py]: except (KeyError, ValueError)

**Review by moznuy — CHANGES_REQUESTED** [review]

File: todo/db.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    done = db.Column(db.Boolean)
+    date = db.Column(db.DateTime)
+
+    def __init__(self, task):
+        self.task = task
+        self.done = False
+        self.date = datetime.datetime.now()
+
+
+db.create_all()
```
Comment (moznuy) [inline todo/db.py]: It's not production safe to run this on every import

File: todo/db.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+def commit_db_changes():
+    db.session.commit()
+
+
+def remove_todo_by_id(todo_id):
+    db.session.delete(get_todo_by_id(todo_id))
+    db.session.commit()
+
+
+def get_todo_list():
```
Comment (moznuy) [inline todo/db.py:37]: When >1 always provide ordering

**Review by moznuy — CHANGES_REQUESTED** [review]

File: todo/db.py (hunk 2 above)
Comment (moznuy) [reply todo/db.py]: You can do smth like this
```
heroku run python3
>>> from app import db
>>> db.create_all()
```

Comment (D1M0H41K) [reply todo/db.py]: from todo import db
db.db.create_all()

File: todo/__init__.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://ieckcdcpjmjedp:b5d46d6d2b1b5f5e0cc74a7f6172a96cb08bbd623887ea082ededb2051d03d17@ec2-52-30-81-192.eu-west-1.compute.amazonaws.com:5432/db6gac5b4cq0gb'
```
Comment (moznuy) [inline todo/__init__.py]: os.environ['DATABASE_URL']

**Review by moznuy — CHANGES_REQUESTED** [review]

**Review by moznuy — APPROVED** [review]

**Pull request merged**